### PR TITLE
Audit navigation guards

### PR DESCRIPTION
**Pull request by shea786**
Added the ability to guard the moderator pages with `moderatorOnly` meta tag against the route

### CODE REVIEW

**Comment by szainmehdi** [comment]
Can you take into account the changes in #195 (merge in develop) and update this so I can can approve it?

**Comment by shea786** [comment]
Requested changes made @szainmehdi

File: web/src/components/auth/UserMenu.vue [hunk 1]
```diff
@@ -214,6 +214,7 @@ export default class UserMenu extends Vue {
   async logout() {
     await this.$store.dispatch(AuthActions.Logout);
     this.open = false;
+    this.$router.push('/');
```
Comment (szainmehdi) [inline web/src/components/auth/UserMenu.vue:217]: Not sure if we always want to do this. It’s probably fine for now but expect this to be reported as a bug eventually

Comment (shea786) [reply web/src/components/auth/UserMenu.vue:217]: Yeah, what was happening was that the `moderator` logged out but still they were on the moderator dashboard so i decided to redirect on logout however that can be changed

**Pull request merged**